rct229/rules/section22/section22rule32.py

```diff
@@ -0,0 +1,93 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.data_fns.table_G3_5_3_fns import table_G3_5_3_lookup
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import HVAC_SYS
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.utils.assertions import getattr_
+from rct229.utils.std_comparisons import std_equal
+
+CHILLER_PART_LOAD_EFFICIENCY_METRIC = schema_enums[
+    "ChillerPartLoadEfficiencyMetricOptions"
+]
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_7,
+    HVAC_SYS.SYS_8,
+    HVAC_SYS.SYS_11_1,
+    HVAC_SYS.SYS_11_2,
+    HVAC_SYS.SYS_12,
+    HVAC_SYS.SYS_13,
+    HVAC_SYS.SYS_7B,
+    HVAC_SYS.SYS_8B,
+    HVAC_SYS.SYS_11B,  # TODO: Change to SYS_11_1B??
+    HVAC_SYS.SYS_12B,
+    HVAC_SYS.SYS_13B,  # TODO: Remove??
```

Comment:
As you already noted, ```SYS_11_1B``` is correct and ```13B``` should be removed.

Reply:
I've made the corrections